lightblue-latest/docker-compose.yml

```diff
@@ -13,3 +11,10 @@ mongodb:
   ports:
    - "28017:28017"
    - "27017:27017"
+applications:
+  image: lightblue/applications
+  command: /opt/jbossas7/bin/standalone.sh -b 0.0.0.0
```

Comment:
I don't think the problem would be the binding 0.0.0.0 but the docker networking configuration which creates a virtual interface (with its own IP) and it can bridge the traffic between the different network interfaces https://docs.docker.com/articles/networking/